src/common/common.js

```diff
@@ -516,14 +516,14 @@ const MessageHandler = (function () {// eslint-disable-line no-unused-vars
             elMessage.textContent = localizedString;
         }
 
-        if (elMessage.classList.contains("dismissible")) {
+        if (typeof args[1] === "boolean" && args[1]) {
```

Comment:
This needs to be adjusted in the JSDoc, also in all functions, which call `showMessage`.

Reply:
Also better use a const here for code style reasons (who knows what arg1 is, otherwise?). (or, better, [shift](https://github.com/rugk/offline-qr-code/pull/64/commits/5555d9ef24c2f50d68da0cf28bb9dc91973b654e#diff-ccdc4da75e7ac1061a8cc0851e3c7122R504) as I've done before…)

Also this codes likely needs to be moved some lines up, because [in line 515](https://github.com/rugk/offline-qr-code/pull/64/commits/5555d9ef24c2f50d68da0cf28bb9dc91973b654e#diff-ccdc4da75e7ac1061a8cc0851e3c7122R515) I already use _all_ other arguments for the translation of the message – so this would include the boolean…